tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"target": "ES2020",
+		"target": "ES2021",
```

Comment:
**note:** Change the target to allow using newer JS API (in particular `String.prototype.replaceAll`). Node 18, which we use, fully supports ES2021.